$NetBSD: patch-configure,v 1.3 2013/07/29 10:38:54 wiz Exp $

Test guile-www by using its http module, the main module is
removed since 2.37.

Fix unportable test(1) construct.

https://bugzilla.gnome.org/show_bug.cgi?id=686830

--- configure.orig	2013-04-20 21:07:33.000000000 +0000
+++ configure
@@ -21347,10 +21347,10 @@ fi
 
 
 
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking if (www main) is available" >&5
-$as_echo_n "checking if (www main) is available... " >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking if (www http) is available" >&5
+$as_echo_n "checking if (www http) is available... " >&6; }
 
-  $GUILE -c "(use-modules (www main)) (exit ((lambda () 0)))" > /dev/null 2>&1
+  $GUILE -c "(use-modules (www http)) (exit ((lambda () 0)))" > /dev/null 2>&1
   gnc_have_guile_www=$?
 
 	  if test "$gnc_have_guile_www" = "0" ; then gnc_have_guile_www=yes ; else gnc_have_guile_www=no ; fi
@@ -21807,7 +21807,7 @@ else
   CUSTOM_GNC_DBD_DIR_FALSE=
 fi
 
- if test "x$ac_cv_header_dbi_dbi_h" == xyes; then
+ if test "x$ac_cv_header_dbi_dbi_h" = xyes; then
   WITH_DBI_TRUE=
   WITH_DBI_FALSE='#'
 else
